src/evict/evict_page.c

```diff
@@ -94,6 +94,85 @@ __wt_page_release_evict(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t flags)
     return (ret);
 }
 
+/*
+ * __evict_stats_update --
+ *     Update the stats of eviction.
+ *
+ */
+static inline void
+__evict_stats_update(
+  WT_SESSION_IMPL *session, bool success, bool clean_page, bool force_evict_hs, uint32_t flags)
```

Comment:
The set of parameters here is unappealing. My usual guide is that any more than two boolean parameters should probably have a flags parameter instead. It's too easy to get multiple bool args mixed up.
It looks like flags is only used to check for urgent eviction, so it could be one of the function flags as well.

i.e: this would become something like:

```
#define WT_EVICT_STATS_CLEAN 0x01
#define WT_EVICT_STATS_FORCE_HS 0x02
#define WT_EVICT_STATS_SUCCESS 0x04
#define WT_EVICT_STATS_URGENT 0x08
static void 
__evict_stats_update(WT_SESSION_IMPL *session, uint32_t flags)
```